ci/angha_50_test_settings.json

```diff
@@ -1,4 +1,5 @@
 {
+    "timeout.seconds": "600",
```

Comment:
A few tests were timing out. We really shouldn't need this since each program contains a single function as far as I can tell.

I wonder if this is related to the perf degradation that happened when that liveness refactor got merged in @2over12.

Reply:
Turns out I just had to use the `--jobs` flag I added to `lifting-tools-ci`. No problem.